presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/Util.java

```diff
@@ -14,8 +14,11 @@
 package com.facebook.presto.sql.planner.iterative.rule;
```

Comment:
I was expecting Maciej to review this PR, and he is already familiar with this function from the previous PRs, so I didn't take the time to copy the commit message. I'm trying to actually work through the 50-or-so new rules that migrating PruneUnreferencedOutputs will create.